developer/local/utils.sh

```diff
@@ -54,14 +54,10 @@ setupTraps() {
 
 detect_container_engine() {
     CONTAINER_ENGINE="${CONTAINER_ENGINE:-}"
-    ALLOW_ROOTLESS="${ALLOW_ROOTLESS:-false}"
     if [[ -n "${CONTAINER_ENGINE}" ]]; then
       return
     fi
-    CONTAINER_ENGINE="sudo podman"
-    if [[ "${ALLOW_ROOTLESS}" == "true" ]]; then
-        CONTAINER_ENGINE=podman
-    fi
```

Comment:
Since utils.sh is being used by local/kcp, we need to have this block where it uses sudo podman.

Reply:
Fixed.